src/lib/uTop.cppo.ml

```diff
@@ -624,6 +630,37 @@ let () =
     (Toploop.Directive_none
        (fun () -> print_endline (Sys.getcwd ())))
 
+let () =
+  Hashtbl.add Toploop.directive_table "utop_stash"
+    (Toploop.Directive_string
+      (fun fname ->
+        let _ :: entries = LTerm_history.contents stashable_session_history in
+        (* getting and then reversing the entries instead of using
+         * [LTerm_history.save] because the latter has to be integrated
+         * into an LWT program flow. *)
```

Comment:
@diml I apologize, I'm not too familiar with collaborating on projects that use LWT, and I just assumed that having more than one `Lwt_main.run` in an application was poor form. You are right, however, that escaped newlines are not the desired behaviour here.